src/state/slices/marketDataSlice/selectors.ts

```diff
@@ -15,6 +15,12 @@ export const selectMarketDataById = createCachedSelector(
   (marketData, assetId: CAIP19) => marketData[assetId],
 )((_marketData, assetId: CAIP19): CAIP19 => assetId)
 
+export const selectMarketDataUnavailableByAssetId = createSelector(
```

Comment:
Not a fan of adding a selector to check for the *unavailability* - I'd rather check for the presence and check for the falsy value of the returned value from selector.